katello/katello-backup

```diff
@@ -54,6 +54,17 @@ def run_cmd(command, exit_codes=[0])
   end
 end
 
+def create_directories(directory)
+  @dir << "/" unless @dir.end_with? "/"
+  @dir << "katello-backup-" + DateTime.now.strftime('%Y%m%d%H%M%S')
```

Comment:
Not really related to this PR, but is there any reason not to use
`@dir = File.join(directory, "katello-backup-" + DateTime.now.strftime('%Y%m%d%H%M%S'))`
here?